test/helpers.py

```diff
@@ -110,7 +110,7 @@ def build_ping_frame(self, ping_data, flags=None):
         return f
 
     def build_goaway_frame(self,
-                           last_stream_id,
+                           last_stream_id=0,
```

Comment:
Can we leave this passed explicitly? I know it's an annoyance in the test, but I'd rather have the explicitness.